feature-pack/src/main/resources/modules/system/layers/base/org/slf4j/ext/main/module.xml

```diff
@@ -1,40 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-
-<!--
-  ~ JBoss, Home of Professional Open Source.
-  ~ Copyright 2010, Red Hat, Inc., and individual contributors
-  ~ as indicated by the @author tags. See the copyright.txt file in the
-  ~ distribution for a full listing of individual contributors.
-  ~
-  ~ This is free software; you can redistribute it and/or modify it
-  ~ under the terms of the GNU Lesser General Public License as
-  ~ published by the Free Software Foundation; either version 2.1 of
-  ~ the License, or (at your option) any later version.
-  ~
-  ~ This software is distributed in the hope that it will be useful,
-  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
-  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
-  ~ Lesser General Public License for more details.
-  ~
-  ~ You should have received a copy of the GNU Lesser General Public
-  ~ License along with this software; if not, write to the Free
-  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
-  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
-  -->
-
-<module xmlns="urn:jboss:module:1.5" name="org.slf4j.ext">
-
-    <properties>
-        <property name="jboss.api" value="deprecated"/>
```

Comment:
This was deprecated in EAP 7.1.0.
For EAP 6.4.0 and 7.0.0 it was public module.

This cannot be removed now

Reply:
ah, it's move of slf4j stuff to wf-core, not total removal. Ignore my comment please.

https://github.com/wildfly/wildfly-core/pull/3182/files#diff-0267876148955d4bc36a55f5c9fae951R25